src/tools/perf/ucp_tests.cc

```diff
@@ -108,6 +108,27 @@ class ucp_perf_test_runner {
         return UCS_OK;
     }
 
+    void send_tag_complete()
```

Comment:
`UCS_F_ALWAYS_INLINE` ? pls add something symmetric like `send_tag_add` and move to `private:` section both (or remove this one).

Reply:
ok